### PR TITLE
Fix mockStore on native

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] Fixes metaDataMap missing from a dumb component
- [x] Don't make all the components over and over again
- [x] Support mockStore in native

@keybase/react-hackers

### CODE REVIEW

File: shared/dev/dumb-sheet/render.native.js [hunk 1]
```diff
@@ -64,63 +68,95 @@ class DumbSheetRender extends Component<void, Props, any> {
     return this.props.autoIncrement ? this.renderIncrement() : this.renderSingle()
   }
 
-  _getComponents (filter: ?string) {
-    const components = []
-    const componentsOnly = []
-    const parentPropsOnly = []
+  _getTotal (filter: ?string) {
```
Comment (chrisnojima) [inline shared/dev/dumb-sheet/render.native.js]: we often call _getComponents to just get the count.

File: shared/dev/dumb-sheet/render.native.js [hunk 2]
```diff
@@ -64,63 +68,95 @@ class DumbSheetRender extends Component<void, Props, any> {
...
 
       const map = dumbComponentMap[key]
       const Component = map.component
+
       Object.keys(map.mocks).forEach((mockKey, idx) => {
-        const mock = {...map.mocks[mockKey]}
-        const parentProps = mock.parentProps
-        mock.parentProps = undefined
-
-        components.push(
```
Comment (chrisnojima) [inline shared/dev/dumb-sheet/render.native.js]: we only ever want one component so lets not keep making each one

File: shared/dev/dumb-sheet/render.native.js [hunk 3]
```diff
@@ -64,63 +68,95 @@ class DumbSheetRender extends Component<void, Props, any> {
...
-    return <Box>{sub}</Box>
+    const {component, mockStore} = this._getComponent(null, this.state.testIndex)
+
+    this._updateMockStore(mockStore)
+    console.log('test render idx', this.state.testIndex, component)
+    return <Box>{this._makeStoreWrapper(component)}</Box>
+  }
+
+  _updateMockStore (mockStore) {
+    if (mockStore) {
```
Comment (chrisnojima) [inline shared/dev/dumb-sheet/render.native.js]: Do what desktop does. On RN you'll get a setState message but its likely not fixing this as this is dev-only and likely a rabbit hole

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
now includes: https://github.com/keybase/client/pull/6351

**Comment by chrisnojima** [comment]
- [ ] Different fix for android emoji

**Comment by keybase-ci-visdiff** [comment]
The commits b6590351ef7d...9fd087735746 introduce visual changes on linux. <details><summary>:mag_right: 14 changed</summary>

 * Changed: **You_Rekey-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/You_Rekey-Normal-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/You_Rekey-Normal-9fd087735746.png)  
   You_Rekey-Normal comparison
 * Changed: **Chat_Participant_Rekey-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Participant_Rekey-Normal-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Participant_Rekey-Normal-9fd087735746.png)  
   Chat_Participant_Rekey-Normal comparison
 * Changed: **Chat_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_List-Normal-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_List-Normal-9fd087735746.png)  
   Chat_List-Normal comparison
 * Changed: **Chat_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_List-Empty-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_List-Empty-9fd087735746.png)  
   Chat_List-Empty comparison
 * Changed: **Chat_Conversations_List-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-You_Rekey_Selected-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-You_Rekey_Selected-9fd087735746.png)  
   Chat_Conversations_List-You_Rekey_Selected comparison
 * Changed: **Chat_Conversations_List-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-You_Rekey-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-You_Rekey-9fd087735746.png)  
   Chat_Conversations_List-You_Rekey comparison
 * Changed: **Chat_Conversations_List-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Selected_Normal-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Selected_Normal-9fd087735746.png)  
   Chat_Conversations_List-Selected_Normal comparison
 * Changed: **Chat_Conversations_List-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Selected_Muted-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Selected_Muted-9fd087735746.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Selected_Muted.png)
 * Changed: **Chat_Conversations_List-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Part_Rekey_Selected-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Part_Rekey_Selected-9fd087735746.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Part_Rekey_Selected.png)
 * Changed: **Chat_Conversations_List-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Part_Rekey-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Part_Rekey-9fd087735746.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Part_Rekey.png)
 * Changed: **Chat_Conversations_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Normal-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Normal-9fd087735746.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Normal.png)
 * Changed: **Chat_Conversations_List-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Long_Top-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Long_Top-9fd087735746.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Long_Top.png)
 * Changed: **Chat_Conversations_List-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Long_Bottom-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Long_Bottom-9fd087735746.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Long_Bottom.png)
 * Changed: **Chat_Conversations_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Empty-b6590351ef7d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Empty-9fd087735746.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490369898250-b6590351ef7d-9fd087735746-linux/Chat_Conversations_List-Empty.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 93eb2b461a80...cf232e464ea2 introduce visual changes on linux. <details><summary>:mag_right: 14 changed</summary>

 * Changed: **You_Rekey-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/You_Rekey-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/You_Rekey-Normal-cf232e464ea2.png)  
   You_Rekey-Normal comparison
 * Changed: **Chat_Participant_Rekey-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Participant_Rekey-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Participant_Rekey-Normal-cf232e464ea2.png)  
   Chat_Participant_Rekey-Normal comparison
 * Changed: **Chat_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_List-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_List-Normal-cf232e464ea2.png)  
   Chat_List-Normal comparison
 * Changed: **Chat_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_List-Empty-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_List-Empty-cf232e464ea2.png)  
   Chat_List-Empty comparison
 * Changed: **Chat_Conversations_List-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-You_Rekey_Selected-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-You_Rekey_Selected-cf232e464ea2.png)  
   Chat_Conversations_List-You_Rekey_Selected comparison
 * Changed: **Chat_Conversations_List-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-You_Rekey-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-You_Rekey-cf232e464ea2.png)  
   Chat_Conversations_List-You_Rekey comparison
 * Changed: **Chat_Conversations_List-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Selected_Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Selected_Normal-cf232e464ea2.png)  
   Chat_Conversations_List-Selected_Normal comparison
 * Changed: **Chat_Conversations_List-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Selected_Muted-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Selected_Muted-cf232e464ea2.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Selected_Muted.png)
 * Changed: **Chat_Conversations_List-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Part_Rekey_Selected-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Part_Rekey_Selected-cf232e464ea2.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Part_Rekey_Selected.png)
 * Changed: **Chat_Conversations_List-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Part_Rekey-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Part_Rekey-cf232e464ea2.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Part_Rekey.png)
 * Changed: **Chat_Conversations_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Normal-cf232e464ea2.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Normal.png)
 * Changed: **Chat_Conversations_List-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Long_Top-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Long_Top-cf232e464ea2.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Long_Top.png)
 * Changed: **Chat_Conversations_List-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Long_Bottom-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Long_Bottom-cf232e464ea2.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Long_Bottom.png)
 * Changed: **Chat_Conversations_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Empty-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Empty-cf232e464ea2.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370043610-93eb2b461a80-cf232e464ea2-linux/Chat_Conversations_List-Empty.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 93eb2b461a80...90399144141b introduce visual changes on linux. <details><summary>:mag_right: 15 changed</summary>

 * Changed: **unlock_folders-Single** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/unlock_folders-Single-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/unlock_folders-Single-90399144141b.png)  
   unlock_folders-Single comparison
 * Changed: **You_Rekey-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/You_Rekey-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/You_Rekey-Normal-90399144141b.png)  
   You_Rekey-Normal comparison
 * Changed: **Chat_Participant_Rekey-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Participant_Rekey-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Participant_Rekey-Normal-90399144141b.png)  
   Chat_Participant_Rekey-Normal comparison
 * Changed: **Chat_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_List-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_List-Normal-90399144141b.png)  
   Chat_List-Normal comparison
 * Changed: **Chat_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_List-Empty-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_List-Empty-90399144141b.png)  
   Chat_List-Empty comparison
 * Changed: **Chat_Conversations_List-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-You_Rekey_Selected-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-You_Rekey_Selected-90399144141b.png)  
   Chat_Conversations_List-You_Rekey_Selected comparison
 * Changed: **Chat_Conversations_List-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-You_Rekey-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-You_Rekey-90399144141b.png)  
   Chat_Conversations_List-You_Rekey comparison
 * Changed: **Chat_Conversations_List-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Selected_Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Selected_Normal-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Selected_Normal.png)
 * Changed: **Chat_Conversations_List-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Selected_Muted-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Selected_Muted-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Selected_Muted.png)
 * Changed: **Chat_Conversations_List-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Part_Rekey_Selected-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Part_Rekey_Selected-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Part_Rekey_Selected.png)
 * Changed: **Chat_Conversations_List-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Part_Rekey-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Part_Rekey-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Part_Rekey.png)
 * Changed: **Chat_Conversations_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Normal-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Normal-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Normal.png)
 * Changed: **Chat_Conversations_List-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Long_Top-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Long_Top-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Long_Top.png)
 * Changed: **Chat_Conversations_List-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Long_Bottom-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Long_Bottom-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Long_Bottom.png)
 * Changed: **Chat_Conversations_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Empty-93eb2b461a80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Empty-90399144141b.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490370936336-93eb2b461a80-90399144141b-linux/Chat_Conversations_List-Empty.png)
</summary>

**Pull request merged**